### ISSUE TITLE
Use explicit namespacing in reports

### ISSUE THREAD

**Issue by jamesmbaazam** [epiverse-trace/episoap#91]
As the purpose of this package is to illustrate useful pipelines (and packages) for achieving end-to-end tasks, I wonder if it might not be better to use explicit namespacing (i.e., `package::some_function()`) so the user knows exactly where the functions are coming from.

**Comment by avallecam** [comment]
I totally agree, this is sth that I can also consider doing at the beginning of any material.
